dsl/pace/dsl/dace/utils.py

```diff
@@ -12,14 +13,21 @@
 
 # Rough timer & log for major operations of DaCe build stack
 class DaCeProgress:
```

Comment:
Oh, this is our logger. Is it used throughout pace, or is there another logger alongside it?

Reply:
We don't have a central logger class, we just get the default logger going. We should have a global class probably, out of scope though

Reply:
Updated w/ a small description